apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisQueryFormModal.vue

```diff
@@ -106,6 +118,22 @@ watch(() => state.proxyVisible, (visible) => {
         state.showValidation = false;
     }
 });
+
+onMounted(async () => {
+    try {
+        const { status, response } = await costQuerySetFetcher({
+            query: {
+                filter: [{ k: 'user_id', v: store.state.user.userId, o: 'eq' }],
+                only: ['name'],
+            },
+        });
+        if (status === 'succeed' && response?.results) {
+            state.existingCostQuerySetNameList = [...state.existingCostQuerySetNameList, ...response.results.map((query) => query.name)];
```

Comment:
issue: What if the modal is popped up several times? Then previous existingCostQuerySetNameList will be accumulated.
We don't know that existingCostQuerySetNameList is initiated one or used one.
I think it's hard to debug if the code works differently depends on the context.
I recommend to separate the managedCostQuerySetIdList and existingCostQuerySetNameList, and use merged list for check.

Reply:
Thank for catching issue that I missed. I rewrite that code. Check plz!